src/pages/createCostModelWizard/addPriceList.tsx

```diff
@@ -0,0 +1,100 @@
+import {
+  Button,
+  FormGroup,
+  FormSelect,
+  FormSelectOption,
+  InputGroup,
+  InputGroupText,
+  TextInput,
+  Title,
+  TitleSize,
+} from '@patternfly/react-core';
+import React from 'react';
+import { CostModelContext } from './context';
+import { MetricType, unit } from './metric.utils';
+
+const AddPriceList = () => {
+  return (
+    <CostModelContext.Consumer>
+      {({ priceListCurrent, updateCurrentPL, submitCurrentPL }) => {
+        return (
+          <>
+            <Title size={TitleSize.xl}>Set a price list</Title>
+            <Title size={TitleSize.md}>
+              To create a price list, select a metric and fill in the values for
+              its tier(s). You can set one or more rates in a price list.
+            </Title>
+            <FormGroup label="Metric" fieldId="metric-selector">
+              <FormSelect
+                value={priceListCurrent.metric}
+                onChange={(value: string) => updateCurrentPL('metric', value)}
+                aria-label="select metric"
+                id="metric-selector"
+              >
+                <FormSelectOption isDisabled value="" label="Select..." />
+                <FormSelectOption value="cpu" label="CPU" />
+                <FormSelectOption value="memory" label="Memory" />
+                <FormSelectOption value="storage" label="Storage" />
+              </FormSelect>
+            </FormGroup>
+            {priceListCurrent.metric !== '' && (
+              <FormGroup label="Measurement" fieldId="measurement-selector">
+                <FormSelect
+                  value={priceListCurrent.measurement}
+                  onChange={(value: string) =>
+                    updateCurrentPL('measurement', value)
+                  }
+                  aria-label="select measurement"
+                  id="measurement-selector"
+                >
+                  <FormSelectOption isDisabled value="" label="Select..." />
+                  <FormSelectOption
+                    value="request"
+                    label={`Request (${unit(
+                      priceListCurrent.metric as MetricType
+                    )})`}
+                  />
+                  <FormSelectOption
+                    value="usage"
+                    label={`Usage (${unit(
+                      priceListCurrent.metric as MetricType
+                    )})`}
+                  />
+                </FormSelect>
+              </FormGroup>
+            )}
+            {priceListCurrent.measurement !== '' && (
+              <FormGroup label="Rate" fieldId="rate-input-box">
+                <InputGroup>
+                  <TextInput
+                    type="number"
+                    aria-label="rate in dollars"
+                    id="rate-input-box"
+                    value={priceListCurrent.rate}
+                    onChange={(value: string) => updateCurrentPL('rate', value)}
+                  />
+                  <InputGroupText>$</InputGroupText>
```

Comment:
The $ symbol appears outside the text field, surrounded by a border. The mockup shows something more like this:

<img width="147" alt="Screen Shot 2019-08-16 at 10 39 52 AM" src="https://user-images.githubusercontent.com/17481322/63175749-ab749780-c012-11e9-84cf-fb495bc35f52.png">